src/fr/fr.json

```
@@ -61,6 +61,8 @@
     "demo": "/fr/demo",
     "releaseNotes": "https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md",
     "privacy": "/fr/avis-de-confidentialite/",
+    "termsofuse": "/fr/conditions-dutilisation",
```

same comment as above - would be nice to keep the case consistent
```suggestion
    "termsOfUse": "/fr/conditions-dutilisation",
```